exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -32,7 +32,7 @@
 import java.util.logging.Logger;
 import javax.annotation.concurrent.ThreadSafe;
 
-/** Exports spans to Jaeger via gRPC, using Jaeger's protobuf model. */
+/** Exports spans to Jaeger via gRPC, using Jaeger's protobuf data. */
```

Comment:
Is this change intentional?

Reply:
reverted for the moment the change to move classes to data. this was clearly not intentional but I initially moved things to model then to data and idea thought is good to change model->data everywhere :)